Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version:5.5
```

Comment:
Could you please format this file? The indentation isn't consistent. I've been using this with swift-format for the rest of the project, for reference:

```json
{
	"indentation": {
		"tabs": 1
	},
	"indentSwitchCaseLabels": true,
	"lineLength": 10000,
	"tabWidth": 4
}
```

I should probably put this in the repo, to be honest…

Reply:
Awesome, I:

 - Added a config and readme about this
 - Formatted Package.swift